conda_dependencies.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
 # Although numpy 1.18.5 & 1.23.4 both worked with python 3.8.10, 3.8.13, 3.8.15, 3.8.16
     - matplotlib # tested versions: 3.6.1, 3.6.2, 3.6.3
     # - pycairo
-    - networkx==2.3
+    - networkx==2.5
```

Comment:
Thanks! I'm guessing it probably has something to do with your python version. Networkx is a little finicky. To avoid breaking other things/potential incompatibility with other setups, I would add "-network==2.5" as a comment in the yaml file for other users and included point this issue out in the installation guide.

Reply:
Just to remember that you should not install (avoid) networkx=2.4 which is unstable and lot of bugs.

Reply:
Thanks. I have also added a note about ` networkx == 2.5` in the docs. Thanks for the additional info on `2.4`

Reply:
`
    - networkx==2.3
# if nextworkx==2.3 fails, try networkx==2.5    
    # - networkx==2.5
`
Since `2.3` works in the conda env, could you add the above in the yaml so I can merge.